src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const [...inputs] = document.querySelectorAll('input');
```

Comment:
```suggestion
const inputs = document.querySelectorAll('input');
```
это итерируемый объект